srcpkgs/gedit/template

```diff
@@ -1,34 +1,34 @@
 # Template file for 'gedit'
 # keep major version in sync with gedit-plugins
 pkgname=gedit
-version=44.2
-revision=2
+version=46.1
+revision=1
 build_helper="gir"
 build_style=meson
 pycompile_dirs="usr/lib/gedit/plugins"
 configure_args="-Dgtk_doc=false"
-hostmakedepends="itstool pkg-config glib-devel gdk-pixbuf perl gettext vala"
-makedepends="gsettings-desktop-schemas-devel gspell-devel gtksourceview4-devel
- libpeas-devel python3-gobject-devel amtk-devel tepl-devel"
+hostmakedepends="itstool pkg-config glib-devel gdk-pixbuf perl gettext
+ gtk-update-icon-cache desktop-file-utils"
+makedepends="gsettings-desktop-schemas-devel gspell-devel libgedit-gtksourceview-devel
+ libpeas-devel python3-gobject-devel libgedit-amtk-devel tepl-devel"
 depends="desktop-file-utils gsettings-desktop-schemas iso-codes"
 short_desc="Text editor for GNOME"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Apps/Gedit"
-changelog="https://gitlab.gnome.org/GNOME/gedit/-/raw/gedit-44/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/gedit/-/raw/46.1/NEWS"
```

Comment:
```suggestion
changelog="https://gitlab.gnome.org/GNOME/gedit/-/raw/${version}/NEWS"
```

If this is consistent upstream (i.e., they always create a branch with the version) you can use a variable, `xbps-src` will turn that into a real link at build time.